google-newtab/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Chrome Extension</title>
+    <script defer src="js/main.js"></script>
```

Comment:
I can tell that you put a lot of thought into organizing your code, and i absolutely applaud you on that! However, it's best practice to put the script tag as the last thing in the body tag. The reason is so that the html loads completely before any script is run. If the html is not loaded all the way and the script loads then you could end up with broken things.

Reply:
Thank you for that feedback. I will start doing that from now on!!